src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### Changed Rules
```

Comment:
This is potentially wrong? Maybe it should've been New Rules here instead of Changed Rules